packages/playground/src/sidebar/runtime.ts

```diff
@@ -165,6 +165,11 @@ function rewireLoggingToElement(
       const prefix = `[<span class="log-${name}">${id}</span>]: `
       const eleContainerLog = eleOverflowLocator()
       allLogs.push(`${prefix}${output}<br>`)
+
+      if (output.includes("Unexpected token 'export'")) {
+        allLogs.push(`[<span class="log-warn">WRN</span>]: Please consider changing your Module to \"CommonJS\" in the TS Config settings`)
```

Comment:
It might be good to provide some additional context on why this change would be useful. Also I don't think the template quotes are necessary here?
```suggestion
        allLogs.push('[<span class="log-warn">WRN</span>]: Tip: Change the Module setting to "CommonJS" in TS Config settings to allow top-level exports to work in the Playground')
```